packages/design-system/src/styles/components/_Table.scss

```diff
@@ -1,13 +1,18 @@
 @import '../settings/index.scss';
 
+$table-header-background-color: $color-gray-lightest !default;
+$table-stripe-background-color: $color-gray-lightest !default;
+$table-stripe-header-background-color: $color-primary-alt-lightest !default;
```

Comment:
We shouldnt use `$color-primary-alt-lightest` here I dont believe, unless we have designers consensus on it. Most designers said that the current gray background is ok because it is also bold, and there are other stylings to explore like increasing the border width. Since there is no consensus, I would just keep it as is.

Reply:
I understand that in that CMS post someone said that existing tables had that style, but those tables weren't even using the CMSDS table styles. And even if they were, we shouldn't jump to change our styles just because of that one case.